docs/tutorials/everything-is-an-event.md

```diff
@@ -0,0 +1,125 @@
+# Everything is an event
+
+This guide will show an example of running the new tracee experience we are experimenting with on Kubernetes.
+For more context on this change, please check the dicussion on github: LINK
```

Comment:
1. typo: discussion
2. LINK should be replaced with a proper link

Reply:
Yes, the discussion was not created yet, it is a placeholder